Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -124,6 +125,7 @@ extension ManifestLoaderProtocol {
         fileSystem: FileSystem? = nil,
         diagnostics: DiagnosticsEngine? = nil,
         on queue: DispatchQueue,
+        verbosity: Verbosity = TSCUtility.verbosity,
```

Comment:
I'd rather not have a default set here to make sure the call-site make a conscious decision. For the unit test we can have an short-form via extension